error: An error occurred: Out of range value for column 'phonenumber' at row 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$834J1sLW3e/QytwvdZISXe6iXSAv0MUfbs95QSIMlwuIuIB.Iu/HG","2023-07-13 11:26:27","2023-07-13 11:26:27"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$834J1sLW3e/QytwvdZISXe6iXSAv0MUfbs95QSIMlwuIuIB.Iu/HG","2023-07-13 11:26:27","2023-07-13 11:26:27"],"parent":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$834J1sLW3e/QytwvdZISXe6iXSAv0MUfbs95QSIMlwuIuIB.Iu/HG","2023-07-13 11:26:27","2023-07-13 11:26:27"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async signup.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.signupdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/signup.js:20:18)"}
error: An error occurred: Out of range value for column 'phonenumber' at row 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$fnfEPfuUd8Sg0cS1HJYqKu83G.0bzHWAOt8nOY4PqGp7empL49nX6","2023-07-13 11:31:37","2023-07-13 11:31:37"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$fnfEPfuUd8Sg0cS1HJYqKu83G.0bzHWAOt8nOY4PqGp7empL49nX6","2023-07-13 11:31:37","2023-07-13 11:31:37"],"parent":{"code":"ER_WARN_DATA_OUT_OF_RANGE","errno":1264,"parameters":["Vinay",9618421328,"kvk.9618@gmail.com","$2b$10$fnfEPfuUd8Sg0cS1HJYqKu83G.0bzHWAOt8nOY4PqGp7empL49nX6","2023-07-13 11:31:37","2023-07-13 11:31:37"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Out of range value for column 'phonenumber' at row 1","sqlState":"22003"},"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async signup.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.signupdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/signup.js:20:18)"}
error: An error occurred: notNull Violation: messages.message cannot be null,
notNull Violation: messages.userId cannot be null {"errors":[{"instance":{"createdAt":"2023-07-14T03:59:21.022Z","id":null,"updatedAt":"2023-07-14T03:59:21.022Z"},"message":"messages.message cannot be null","origin":"CORE","path":"message","type":"notNull Violation","validatorArgs":[],"validatorKey":"is_null","validatorName":null,"value":null},{"instance":{"createdAt":"2023-07-14T03:59:21.022Z","id":null,"updatedAt":"2023-07-14T03:59:21.022Z"},"message":"messages.userId cannot be null","origin":"CORE","path":"userId","type":"notNull Violation","validatorArgs":[],"validatorKey":"is_null","validatorName":null,"value":null}],"name":"SequelizeValidationError","stack":"SequelizeValidationError: notNull Violation: messages.message cannot be null,\nnotNull Violation: messages.userId cannot be null\n    at InstanceValidator._validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:50:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async InstanceValidator._validateAndRunHooks (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:60:7)\n    at async InstanceValidator.validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:54:12)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2426:7)\n    at async messages.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.chatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:11:18)"}
error: An error occurred: notNull Violation: groups.group_description cannot be null {"errors":[{"instance":{"createdAt":"2023-07-14T13:05:49.389Z","creator_id":4,"group_id":null,"group_name":"sharpers group","updatedAt":"2023-07-14T13:05:49.389Z"},"message":"groups.group_description cannot be null","origin":"CORE","path":"group_description","type":"notNull Violation","validatorArgs":[],"validatorKey":"is_null","validatorName":null,"value":null}],"name":"SequelizeValidationError","stack":"SequelizeValidationError: notNull Violation: groups.group_description cannot be null\n    at InstanceValidator._validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:50:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async InstanceValidator._validateAndRunHooks (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:60:7)\n    at async InstanceValidator.validate (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/instance-validator.js:54:12)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2426:7)\n    at async groups.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.creategroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:10:18)"}
error: An error occurred: Table 'chat_connect1.groups' doesn't exist {"name":"SequelizeDatabaseError","original":{"code":"ER_NO_SUCH_TABLE","errno":1146,"parameters":["sharpener group",4,"2023-07-14 13:10:24","2023-07-14 13:10:24"],"sql":"INSERT INTO `groups` (`group_id`,`group_name`,`creator_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?);","sqlMessage":"Table 'chat_connect1.groups' doesn't exist","sqlState":"42S02"},"parameters":["sharpener group",4,"2023-07-14 13:10:24","2023-07-14 13:10:24"],"parent":{"code":"ER_NO_SUCH_TABLE","errno":1146,"parameters":["sharpener group",4,"2023-07-14 13:10:24","2023-07-14 13:10:24"],"sql":"INSERT INTO `groups` (`group_id`,`group_name`,`creator_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?);","sqlMessage":"Table 'chat_connect1.groups' doesn't exist","sqlState":"42S02"},"sql":"INSERT INTO `groups` (`group_id`,`group_name`,`creator_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async groups.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.creategroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:10:18)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Unknown column 'groupGroupId' in 'field list' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","sqlMessage":"Unknown column 'groupGroupId' in 'field list'","sqlState":"42S22"},"sql":"SELECT `id`, `message`, `userId`, `createdAt`, `updatedAt`, `groupGroupId` FROM `messages` AS `messages` WHERE `messages`.`userId` = 4;","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async messages.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:25:19)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:24:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:24:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.getchatdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/chat.js:24:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1 {"name":"SequelizeDatabaseError","original":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"parameters":{},"parent":{"code":"ER_PARSE_ERROR","errno":1064,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci'' at line 1","sqlState":"42000"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` ILIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: Unknown column 'signup.collate' in 'where clause' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` LIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"Unknown column 'signup.collate' in 'where clause'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` LIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","sqlMessage":"Unknown column 'signup.collate' in 'where clause'","sqlState":"42S22"},"sql":"SELECT `id`, `username`, `phonenumber`, `email`, `password`, `createdAt`, `updatedAt` FROM `signups` AS `signup` WHERE `signup`.`username` LIKE '%a%' AND `signup`.`collate` = 'utf8_general_ci';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async signup.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.getusersearchdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/usersearch.js:11:18)"}
error: An error occurred: data is not defined {"stack":"ReferenceError: data is not defined\n    at exports.getuserdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/user.js:10:5)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: userDetails is not defined {"stack":"ReferenceError: userDetails is not defined\n    at exports.getuserdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/user.js:13:43)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: connect ETIMEDOUT {"name":"SequelizeConnectionError","original":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"parent":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"stack":"SequelizeConnectionError: connect ETIMEDOUT\n    at ConnectionManager.connect (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:102:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)"}
error: An error occurred: GroupModel is not defined {"stack":"ReferenceError: GroupModel is not defined\n    at exports.getgroupnames (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:25:20)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of undefined (reading 'id') {"stack":"TypeError: Cannot read properties of undefined (reading 'id')\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:28:28\n    at Array.map (<anonymous>)\n    at exports.getgroupnames (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:26:31)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: connect ETIMEDOUT {"name":"SequelizeConnectionError","original":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"parent":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"stack":"SequelizeConnectionError: connect ETIMEDOUT\n    at ConnectionManager.connect (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:102:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ConnectionManager._connect (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)"}
error: An error occurred: Validation error {"errors":[{"instance":{"createdAt":"2023-07-21T07:37:50.037Z","email":"vinay@gmail.com","id":null,"password":"$2b$10$MeKEoUFoB743sy.hbvbAG.6T/BcIYf7hpQyyCfIOwTaMCqIF7JIXC","phonenumber":"9876543210","updatedAt":"2023-07-21T07:37:50.037Z","username":"vinay"},"message":"email must be unique","origin":"DB","path":"email","type":"unique violation","validatorArgs":[],"validatorKey":"not_unique","validatorName":null,"value":"vinay@gmail.com"}],"fields":{"email":"vinay@gmail.com"},"name":"SequelizeUniqueConstraintError","original":{"code":"ER_DUP_ENTRY","errno":1062,"parameters":["vinay","9876543210","vinay@gmail.com","$2b$10$MeKEoUFoB743sy.hbvbAG.6T/BcIYf7hpQyyCfIOwTaMCqIF7JIXC","2023-07-21 07:37:50","2023-07-21 07:37:50"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Duplicate entry 'vinay@gmail.com' for key 'signups.email'","sqlState":"23000"},"parent":{"code":"ER_DUP_ENTRY","errno":1062,"parameters":["vinay","9876543210","vinay@gmail.com","$2b$10$MeKEoUFoB743sy.hbvbAG.6T/BcIYf7hpQyyCfIOwTaMCqIF7JIXC","2023-07-21 07:37:50","2023-07-21 07:37:50"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Duplicate entry 'vinay@gmail.com' for key 'signups.email'","sqlState":"23000"},"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async signup.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.signupdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/signup.js:20:18)"}
error: An error occurred: Validation error {"errors":[{"instance":{"createdAt":"2023-07-21T07:39:07.111Z","email":"vinaykumar@gmail.com","id":null,"password":"$2b$10$XFdulDCIxveDA2NI0kn3du9KO7lc/TKwPGd16WMOUafSAFUP.Umca","phonenumber":"9876543210","updatedAt":"2023-07-21T07:39:07.111Z","username":"vinaykumar"},"message":"email must be unique","origin":"DB","path":"email","type":"unique violation","validatorArgs":[],"validatorKey":"not_unique","validatorName":null,"value":"vinaykumar@gmail.com"}],"fields":{"email":"vinaykumar@gmail.com"},"name":"SequelizeUniqueConstraintError","original":{"code":"ER_DUP_ENTRY","errno":1062,"parameters":["vinaykumar","9876543210","vinaykumar@gmail.com","$2b$10$XFdulDCIxveDA2NI0kn3du9KO7lc/TKwPGd16WMOUafSAFUP.Umca","2023-07-21 07:39:07","2023-07-21 07:39:07"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Duplicate entry 'vinaykumar@gmail.com' for key 'signups.email'","sqlState":"23000"},"parent":{"code":"ER_DUP_ENTRY","errno":1062,"parameters":["vinaykumar","9876543210","vinaykumar@gmail.com","$2b$10$XFdulDCIxveDA2NI0kn3du9KO7lc/TKwPGd16WMOUafSAFUP.Umca","2023-07-21 07:39:07","2023-07-21 07:39:07"],"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","sqlMessage":"Duplicate entry 'vinaykumar@gmail.com' for key 'signups.email'","sqlState":"23000"},"sql":"INSERT INTO `signups` (`id`,`username`,`phonenumber`,`email`,`password`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async signup.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.signupdetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/signup.js:20:18)"}
error: An error occurred: WHERE parameter "group_id" has invalid "undefined" value {"stack":"Error: WHERE parameter \"group_id\" has invalid \"undefined\" value\n    at MySQLQueryGenerator.whereItemQuery (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1743:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1734:25\n    at Array.forEach (<anonymous>)\n    at MySQLQueryGenerator.whereItemsQuery (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1732:35)\n    at MySQLQueryGenerator.getWhereConditions (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2075:19)\n    at MySQLQueryGenerator.selectQuery (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:988:28)\n    at MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:59)\n    at groups.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:47)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:30)"}
error: An error occurred: WHERE parameter "group_id" has invalid "undefined" value {"stack":"Error: WHERE parameter \"group_id\" has invalid \"undefined\" value\n    at MySQLQueryGenerator.whereItemQuery (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1743:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1734:25\n    at Array.forEach (<anonymous>)\n    at MySQLQueryGenerator.whereItemsQuery (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1732:35)\n    at MySQLQueryGenerator.getWhereConditions (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2075:19)\n    at MySQLQueryGenerator.selectQuery (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-generator.js:988:28)\n    at MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:59)\n    at groups.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:47)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:30)"}
error: An error occurred: group.findByID is not a function {"stack":"TypeError: group.findByID is not a function\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Argument passed to findByPk is invalid: [object Object] {"stack":"Error: Argument passed to findByPk is invalid: [object Object]\n    at groups.findByPk (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1219:13)\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Argument passed to findByPk is invalid: [object Object] {"stack":"Error: Argument passed to findByPk is invalid: [object Object]\n    at groups.findByPk (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1219:13)\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Argument passed to findByPk is invalid: [object Object] {"stack":"Error: Argument passed to findByPk is invalid: [object Object]\n    at groups.findByPk (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1219:13)\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: group.findByID is not a function {"stack":"TypeError: group.findByID is not a function\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: group.findById is not a function {"stack":"TypeError: group.findById is not a function\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Argument passed to findByPk is invalid: [object Object] {"stack":"Error: Argument passed to findByPk is invalid: [object Object]\n    at groups.findByPk (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1219:13)\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:78:42)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot access 'username' before initialization {"stack":"ReferenceError: Cannot access 'username' before initialization\n    at exports.addUserToGroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:40:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: data is not defined {"stack":"ReferenceError: data is not defined\n    at exports.addUserToGroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:50:41)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Cannot read properties of null (reading 'id') {"stack":"TypeError: Cannot read properties of null (reading 'id')\n    at exports.creategroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:8:29)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: data is not defined {"stack":"ReferenceError: data is not defined\n    at exports.addUserToGroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:49:41)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: GroupUser is not defined {"stack":"ReferenceError: GroupUser is not defined\n    at exports.addUserToGroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:46:18)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Unknown column 'group_user.groupId' in 'where clause' {"name":"SequelizeDatabaseError","original":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `role`, `createdAt`, `updatedAt`, `group_id` FROM `group_users` AS `group_user` WHERE `group_user`.`groupId` = '1';","sqlMessage":"Unknown column 'group_user.groupId' in 'where clause'","sqlState":"42S22"},"parameters":{},"parent":{"code":"ER_BAD_FIELD_ERROR","errno":1054,"sql":"SELECT `id`, `role`, `createdAt`, `updatedAt`, `group_id` FROM `group_users` AS `group_user` WHERE `group_user`.`groupId` = '1';","sqlMessage":"Unknown column 'group_user.groupId' in 'where clause'","sqlState":"42S22"},"sql":"SELECT `id`, `role`, `createdAt`, `updatedAt`, `group_id` FROM `group_users` AS `group_user` WHERE `group_user`.`groupId` = '1';","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.select (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)\n    at async group_user.findAll (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1140:21)\n    at async exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:88:20)"}
error: An error occurred: groupId is not defined {"stack":"ReferenceError: groupId is not defined\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:81:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: groupId is not defined {"stack":"ReferenceError: groupId is not defined\n    at exports.groupDetails (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:81:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: groupId is not defined {"stack":"ReferenceError: groupId is not defined\n    at exports.groupDeleteUser (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:119:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: groupId is not defined {"stack":"ReferenceError: groupId is not defined\n    at exports.groupDeleteUser (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:119:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: groupId is not defined {"stack":"ReferenceError: groupId is not defined\n    at exports.groupDeleteUser (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:119:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: groupInformation is not defined {"stack":"ReferenceError: groupInformation is not defined\n    at exports.groupDeleteUser (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:128:41)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: groupname is not defined {"stack":"ReferenceError: groupname is not defined\n    at exports.creategroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:14:19)\n    at Layer.handle [as handle_request] (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/layer.js:95:5)\n    at next (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/express/lib/router/route.js:144:13)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/middleware/auth.js:18:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
error: An error occurred: Validation error {"errors":[{"instance":{"createdAt":"2023-07-23T06:40:48.762Z","group_id":2,"id":1,"role":"member","updatedAt":"2023-07-23T06:40:48.762Z"},"message":"PRIMARY must be unique","origin":"DB","path":"PRIMARY","type":"unique violation","validatorArgs":[],"validatorKey":"not_unique","validatorName":null,"value":"1"}],"fields":{"PRIMARY":"1"},"name":"SequelizeUniqueConstraintError","original":{"code":"ER_DUP_ENTRY","errno":1062,"parameters":[1,"member","2023-07-23 06:40:48","2023-07-23 06:40:48",2],"sql":"INSERT INTO `group_users` (`id`,`role`,`createdAt`,`updatedAt`,`group_id`) VALUES (?,?,?,?,?);","sqlMessage":"Duplicate entry '1' for key 'group_users.PRIMARY'","sqlState":"23000"},"parent":{"code":"ER_DUP_ENTRY","errno":1062,"parameters":[1,"member","2023-07-23 06:40:48","2023-07-23 06:40:48",2],"sql":"INSERT INTO `group_users` (`id`,`role`,`createdAt`,`updatedAt`,`group_id`) VALUES (?,?,?,?,?);","sqlMessage":"Duplicate entry '1' for key 'group_users.PRIMARY'","sqlState":"23000"},"sql":"INSERT INTO `group_users` (`id`,`role`,`createdAt`,`updatedAt`,`group_id`) VALUES (?,?,?,?,?);","stack":"Error\n    at Query.run (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/mysql/query.js:52:25)\n    at /media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.insert (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/dialects/abstract/query-interface.js:308:21)\n    at async model.save (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:2490:35)\n    at async group_user.create (/media/vinay/CC4A7C094A7BEE98/ChatConnect/node_modules/sequelize/lib/model.js:1362:12)\n    at async exports.addUserToGroup (/media/vinay/CC4A7C094A7BEE98/ChatConnect/controllers/group.js:44:18)"}
